docker-compose.yml

```diff
@@ -13,7 +13,9 @@ services:
       NB_GRAPH_PORT: 7200
       NB_GRAPH_DB: ${NB_GRAPH_DB:-repositories/my_db}
       NB_RETURN_AGG: ${NB_RETURN_AGG:-true}
+      NB_MIN_CELL_SIZE: ${NB_MIN_CELL_SIZE}
```

Comment:
how about we set a default value here like for most other values that don't default to null/empty?